05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,48 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.container {
+  height:350px;
+  /*this fixes the container height*/
 }
-
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+.berryimage{
+  width:500px;
+  float:left;
 }
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.berryimage img {
+  width:100%;
+  height:100%;
+  /*this fills the whole div with the image */
 }
-
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.berrycontent {
```

Comment:
if both berryimage and berrycontent have the same width and float, could you factor these out?